daphne/utils.py

```diff
@@ -15,11 +15,11 @@ def import_by_path(path):
     return target
 
 
-def header_value(headers, header_name):
+def header_value(headers, header_name) -> str:
     value = headers[header_name]
     if isinstance(value, list):
         value = value[0]
-    return value.decode("utf-8")
+    return value.decode("utf-8") if type(value) is bytes else value
```

Comment:
Do you know why your headers are unicode not bytes? Nobody else has ever complained about this, and I don't want to accept this without knowing why.

Reply:
I have no idea why. I can post my rqeuirements.txt if needed, but I run latest-greatest on an ubuntu 16.10 LXC container with python3.6.